wordpress-mobile/WordPress-iOS
# Updated pods; fixed optional with new Mixpanel API, removed space.

Closes #7076 

Updates a few pods with latest versions. Had to tweak Mixpanel in Swift due to nullability. 

To test:
1. `pod repo update`
2. `pod install`
3. Clean, Build, Run.
4. Verify nothing crashes. Try the 1Password extension on a device.

Needs review: @koke
